constraint/pkg/client/crds/validate.go

```diff
@@ -4,8 +4,9 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client/errors"
```

Comment:
Rename to `clienterrors` for consistency.

Reply:
done